code/datums/cult/emp.dm

```diff
@@ -0,0 +1,27 @@
+/datum/rune/emp
+	name = "EMP rune"
+	desc = "This rune is used to disable electronics in an area around us."
+	rune_flags = HAS_SPECIAL_TALISMAN_ACTION
+
+/datum/rune/emp/do_rune_action(mob/living/user, atom/movable/A)
+	do_emp(user, A, 2)
+
+/datum/rune/emp/do_talisman_action(mob/living/user, var/atom/movable/A)
+	do_emp(user, A, 1)
+	return TRUE
+
+/datum/rune/emp/proc/do_emp(mob/living/user, atom/movable/A, radius = 2)
+	user.say("Ta'gh fara'qha fel d'amar det!")
+	log_and_message_admins("activated an EMP [A].")
+	var/turf/T = get_turf(A)
+	playsound(T, 'sound/magic/Disable_Tech.ogg', 25, 1)
+
+	var/list/ex = list(user) // exclude caster
```

Comment:
shouldn't range() already also get the user? Or might the user be a non-cultist?

Reply:
Range gets everything, this is a list you put in empulse() to make it not affect people in that list